src/word-count/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useCallback } from 'react';
+import { TextArea } from '@ukhomeoffice/react-components';
+import classNames from 'classnames';
+
+export default function WordCount(props) {
+
+    const { value, maxWordCount, error, values, name } = props;
```

Comment:
Following on from this suggestion in https://github.com/UKHomeOffice/asl-pages/pull/1651 
```javascript
{
    //...
    maxWordCount: 100
    validate: [
      { lessThanOrEqualToMaxWordCount: 100 }
    ]
}
```

It might be good to have a single source of truth for the wordCount. Possible implementation:

```suggestion
    const { value, validate: {lessThanOrEqualToMaxWordCount: maxWordCount}, error, values, name } = props;
```
I'm not sure this is the most intuitive way to do it, but it does have the side effect that it will force someone using the component to set the required validator. Thoughts?